README.md

```diff
@@ -54,7 +54,7 @@ require('iswap').setup{
   hl_grey = 'LineNr',
 
   -- Post-operation flashing highlight style,
-  -- either 'simultaneous' or 'sequential', or nil to disable
+  -- either 'simultaneous' or 'sequential', or 'none' to disable
```

Comment:
I like the refactoring you've done, and I realize nil cant be used as an option choice with setmetatable's __index, but how about we make disabling `false` instead of nil?

Reply:
sure, your call

Reply:
done. added it to the example to make clear that it's `false` and not `'false'`, but feel free to change. thnx!